### ISSUE TITLE
[New addon] Stepping in Scratch editor

### ISSUE THREAD

**Issue by BoomerScratch** [ScratchAddons/ScratchAddons#74]
**Describe the new addon**

It allows you to see which blocks currently run in the Scratch editor.

**Link to the new addon**

https://gist.github.com/BoomerScratch/0592337050810244236d69a4267b3913

**Comment by WorldLanguages** [comment]
This "find vm" approach should probably be made by the addon loader so that many addons can access the VM object (and not just this one). Related to #17

**Comment by BoomerScratch** [comment]
@WorldLanguages `addon.ScratchVM`?

**Comment by WorldLanguages** [comment]
@BoomerScratch Yeah, it will probably be `addon.tab.getScratchVM` (returns a promise) and will resolve when it finds the VM.
